app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/IngredientsProductFragment.java

```diff
@@ -73,10 +74,11 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
         }
 
         if(mState != null && product.getIngredientsText() != null) {
-            SpannableStringBuilder txtIngredients = new SpannableStringBuilder(Html.fromHtml(product.getIngredientsText().replace("_","")));
-            txtIngredients = setSpanBoldBetweenTokens(txtIngredients);
-            if(!txtIngredients.toString().substring(txtIngredients.toString().indexOf(":")).trim().isEmpty()) {
-                ingredientsProduct.setText(txtIngredients);
+            //TODO The API doesn't return ingredients text with the _ token. the replace method could be removed
```

Comment:
`Html.fromHtml`doesn't support html `span` tag. Moreover the span tag contains a class attribute which is not supported on android for the html parsing. 

The new version of the API should return directly the value in `<b>` tag